src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,130 @@
+package duke;
+
+import java.io.IOException;
+
+/**
+ * Duke class, your personal assistant.
+ */
+public class Duke {
+    private TaskList taskList = new TaskList();
+    private Parser parser = new Parser();
+
+    /**
+     * Start the Duke.
+     */
+    public void start() {
+        Storage storage = new Storage("data.txt");
+        storage.fillData(taskList);
+
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        UserInterface.print("Hello from\n" + logo);
```

Comment:
Maybe it would be better abstraction if all the code/methods constructing messages to print is also encapsulated into the UserInterface class instead of only using it to hold the print method.

Reply:
This can apply to all parts of the code that use UserInterface.print(String) since it appears very frequently.